src/components/markdown.tsx

```diff
@@ -138,11 +138,77 @@ const OPTIONS: MarkdownToJSX.Options = {
 };
 /* @TODO: Try to improve this typings here. */
 
+const parseTabs = (text: string) => {
+  const regex = /tab\[([^\]]+)\]\[([^\]]+)\]((?:.|[\r\n])*?)(?=(?:tab\[)|$)/g;
+  const lines = text.split(`\n`);
+  const tabs = [];
+  let lastLine = 0;
+  let match;
+
+  while ((match = regex.exec(text)) !== null) {
+    const label = match[1];
+    const range = match[2];
+
+    const [start, end] = range.split(`,`).map(Number);
```

Comment:
To check, but I'm not sure about Number, it may take additional parameter as a second constructor argument, and with that it will be always index. I recommend for being sure do arg => Number.parseInt(arg).